ethereum/deploy/01-beefy.ts

```diff
@@ -21,11 +21,13 @@ module.exports = async ({
         MerkleProof: merkleProofLibrary.address
     },
     log: true,
+    autoMine: true,
```

Comment:
does this field just get ignored for ropsten deployments? if so, then great

Reply:
yeah :+1: